### ISSUE TITLE
Support for facebook videos

### ISSUE THREAD

**Issue by nightmora** [essence/essence#129]
Its easy to add support for facebook videos?
Thanks

Plase add this on next commit

```
[
		'Facebook' => Essence\Di\Container::unique(function($C) {
			return $C->get('OEmbedProvider')->setEndpoint('https://www.facebook.com/plugins/video/oembed.json/?url=:url');
		}),

		'filters' => ['Facebook' => '~facebook\.com/.*~i']
	];

```

**Comment by felixgirault** [comment]
Hi,

Would you make a pull request for that ?

**Comment by nightmora** [comment]
Ok, thanks.

**Comment by felixgirault** [comment]
Thank you for the PR, I released version 3.4.0 with it ;)

**Comment by nightmora** [comment]
Thank you very much it has been a pleasure


2017-06-03 23:09 GMT+02:00 Félix Girault <notifications@github.com>:
